### PR TITLE
Add add method in bulk

### LINKED ISSUE #21: Implement a method to register tasks in bulk
Before:  It takes time and effort
```bash
/add kamontia
/add chaspy
```

After:
```bash
/add "kamontia" "chaspy"
```

### PR DESCRIPTION
**Pull request by kamontia**
Deploying to `kimetaro-dev3`
  
image

Close #21

### CODE REVIEW

**Comment by kamontia** [comment]
@chaspy Please review and merge

**Comment by chaspy** [comment]
Let's update `README.md` too!

File: kimetaro.py [hunk 1]
```diff
@@ -9,6 +10,8 @@
 
 # Value initialization
 COMMAND_SUFFIX = ''
+PATTERN = r"(?<=\"|')(?!,).*?(?=\"|')"
```
Comment (chaspy) [inline kimetaro.py]: It's better to add a comment that shows the match pattern

Comment (kamontia) [reply kimetaro.py]: Added the comment that explains what keywords match.

**Comment by kamontia** [comment]
image

Please review and merge

**Comment by chaspy** [comment]
Sorry I mean that it's better to update the following section in README.
https://github.com/kamontia/kimetaro/pull/24#issuecomment-488566134

>#### Register(Not implemented)

It's better to write the limitation in the same section if you can

---

```diff
--- /tmp/readme.txt	2019-05-03 19:15:42.000000000 +0900
+++ /tmp/readme.txt.after	2019-05-03 19:17:38.000000000 +0900
@@ -1,6 +1,11 @@
-#### Register(Not implemented)
+#### Register
 
 # ```bash
 !add planA
-!add planB
+!add planA planB 
+!add "planA" "planB"
# ```
+
+NOTE:
+- `"` (Double quotation) and no quotation can not be mixed
+- `'` (Single quotation) not accepted
```

File: kimetaro.py [hunk 2]
```diff
@@ -87,9 +99,27 @@
 @pysnooper.snoop()
 def add(message):
```
Comment (chaspy) [inline kimetaro.py:100]: This function is too complicated.

It's better to write to inform the following
- This function accepts two types of string
  - `"str"...`
  - `str(space)str`
- The (if) condition branches according to the type of string

Comment (kamontia) [reply kimetaro.py:100]: Fixed up.

**Review by chaspy — APPROVED** [review]
LGTM!

**Pull request merged**